frontend/src/App.tsx

```diff
@@ -40,6 +42,12 @@ const RealmRoute: React.FC<RouteComponentProps<{ path?: string }>> = ({ match })
     </APIWrapper>
 );
 
+const PlayerRoute: React.FC<RouteComponentProps<{ id: string }>> = ({ match }) => (
+    <APIWrapper>
+        <PlayerPage id={match.params.id} />
+    </APIWrapper>
```

Comment:
We have this wrapping `APIWrapper` three times in this file already. Looks like someone could design a fancy HOC or sth like that to avoid duplication? Or... can't we put the `APIWrapper` higher in the tree? Doesn't need to happen in this PR tho.

Reply:
In our case it could live higher up, yes. Once we have proper error handling in place, I expect more than one error boundary, though. This kind of simulates that already.

I can pull it up if it bothers you, but yeah, `APIWrapper` is living on borrowed time AFAIAC.

Reply:
yeah no, let's keep it like this for now.